tests/conftest.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Fixtures for github runner charm."""
+
+
+import argparse
+
+
+def pytest_addoption(parser: argparse.ArgumentParser):
+    """Add options to pytest parser."""
+    # mypy cannot find the `addoption` attr.
+    parser.addoption("--path", action="store")  # type: ignore
```

Comment:
Shouldnt "from pytest import Parser" be used?

Reply:
Updated